charts/csi-isilon/templates/controller.yaml

```diff
@@ -310,6 +296,16 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         {{- if not $encrypted }}
+        - name: csi-metadata-retriever
```

Comment:
is `{{ $csiSidecarSuffix }}` no longer required on the name suffix?

Reply:
Not required, since there is always one instance of the container - for the unencrypted set of containers.